### DISCUSSION TITLE
1.1.1 업데이트 후 에디터상 서버 타임아웃 에러

### DISCUSSION THREAD

**Discussion by mhkim0720** [naverz/zepeto-studio-kor#127] [Issue Report (Legacy)]
안녕하세요 

1.1.1 업데이트 후 유니티 에디터에서 아래와 같은 에러가 발생하여 문의 드립니다. 

![image](https://user-images.githubusercontent.com/8485731/147188264-e706e180-4a94-4d11-9ab5-c7e0a6aa20b5.png)

에디터 런타임 중 서버 관련 에러가 뜨면서 유니티 에디터 플레이가 Pause 되는 현상이 발생 합니다 

확인 부탁 드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 오류가 최신 버전 1.1.5 에서도 지속적으로 나타나는지 확인 부탁드립니다.
